File: Atividade1/Atividade1/src/Usuario.java
// DAVI FERNANDES SIMÕES SOARES - 20.01099-0

public class Usuario {
    private String nome;
    protected BemCompartilhado transporte;
    
    public Usuario(String nome, BemCompartilhado transporte) {
        this.nome = nome;
        this.transporte = transporte;
    }

    public String getNome() {
        return nome;
    }

    public BemCompartilhado getTransporte() {
        return transporte;
    }

    @Override
    public String toString() {
        return "Usuario [nome=" + nome + ", transporte=" + transporte + "]";
    }

    public void TrocaVeiculos(/*BemCompartilhado transporte,*/BemCompartilhado tipo2){
        /*if(transporte.getTipo() != tipo2.getTipo()){
            this.transporte = tipo2;
        }                                                   NÃO DEU TEMPO!!
        System.out.println("Não é possível.");
        */
        this.transporte = tipo2;
    }
}

File: AtividadeT4/src/models/usuario.py
# Davi Fernandes Simões Soares (20.01099-0)

class Usuario():
    def __init__(self, nome, email, senha, cpf):
        self._email = email
        self._nome = nome
        self._senha = senha
        self._cpf = cpf

    def get_senha(self):
        return self._senha

    def get_nome(self):
        return self._nome

    def get_email(self):
        return self._email

    def get_cpf(self):
        return self._cpf

    def __str__(self)->str:
        return f'Usuario(nome:{self.get_nome()}, email:{self.get_email()}, senha:{self.get_senha()} cpf:{self.get_cpf()}'